packages/launchpad/src/checklist-item/index.stories.tsx

```diff
@@ -20,40 +22,84 @@ const baseTask = {
 	completed: false,
 	disabled: false,
 };
+const baseArgs = {
```

Comment:
I added this to make sure the items will have the `onClick`. Now if they don't have it they will be an `div` without the hover/focus styles.